src/writer/proposal.ts

```diff
@@ -36,6 +37,42 @@ export const getProposalsCount = async (space, author) => {
   return await db.queryAsync(query, [space, author]);
 };
 
+async function validateSpace(originalSpace: any) {
+  const space = snapshot.utils.clone(originalSpace);
+
+  if (!space) {
+    return Promise.reject('unknown space');
+  }
+
+  if (space.hibernated) {
+    return Promise.reject('space hibernated');
+  }
+
+  if (space?.deleted) return Promise.reject('space deleted, contact admin');
+
+  delete space.flagged;
+  delete space.verified;
+  delete space.id;
+
+  const schemaIsValid: any = snapshot.utils.validateSchema(snapshot.schemas.space, space, {
+    snapshotEnv: SNAPSHOT_ENV
+  });
```

Comment:
Better to handle these also in `validateSpaceSettings ` and reuse same function on both `settings` and `proposal`